server/middleware/community_middleware.go

```diff
@@ -0,0 +1,22 @@
+package middleware
+
+import (
+	"github.com/bokoness/lashon/services"
+	"github.com/gofiber/fiber/v2"
+)
+
+func CommunityMiddleware(c *fiber.Ctx) error {
+	user, err := services.GetUserFromSession(c)
+
+	if err != nil || user == nil {
```

Comment:
```suggestion
	if err != nil {
```
No need, after my changes in `Gorm Errors` PR, if user is not found, it will return an error